### DISCUSSION TITLE
INVOICE API C# - Codice identificativo destinatario/Codice Univoco Ufficio

### DISCUSSION THREAD

**Discussion by Mustino2022** [fattureincloud/api#240] [Q&A] [answered]
Buongiorno,
nell'operazione di Create Issued Document non trovo la proprietà dove impostare il Codice identificativo destinatario/Codice Univoco Ufficio.
In ogni fattura elettronica è necessario indicare il Codice Univoco Ufficio, che rappresenta l’identificativo univoco che consente al Sistema di Interscambio (SDI) di recapitare correttamente la fattura elettronica all’Ufficio destinatario. Il Codice Univoco Ufficio deve essere inserito nella fattura elettronica in corrispondenza dell’elemento del tracciato 1.1.4 denominato “Codice Destinatario”.

Grazie
Saluti

**Comment by emajo** [comment, marked as answer]
Ciao @Mustino2022, il campo relativo al codice destinatario è `entity.ei_code`, in caso si trattasse di una pa, ovvero con codice destinatario di 6 cifre invece che 7 è necessario specificare `entity.type = pa`.

Grazie,
Emanuele.

**Comment by Mustino2022** [comment]
Ciao Emanuele, grazie per la risposta, ma continuo a non trovare il campo in fase di creazione della invoice

![image](https://user-images.githubusercontent.com/117667863/214287433-445932d5-1cc4-497d-a14f-3f09b5e27084.png)

mentre in fase di lettura del documento Get Issued Document il campo è il seguente **ei_code**?

![image](https://user-images.githubusercontent.com/117667863/214287958-83beee9f-c87e-455b-8a0f-0b777374b039.png)

**Reply by emajo** [reply]
Si, scusami il campo corretto è `entity.ei_code`, ho corretto la risposta sopra.

**Comment by Mustino2022** [comment]
Ciao Emanuele,
ora mi è tutto chiaro, grazie per la collaborazione.
Saluti
